src/main/java/sogorae/auth/support/JwtProvider.java

```diff
@@ -0,0 +1,64 @@
+package sogorae.auth.support;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.security.Keys;
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+import javax.crypto.SecretKey;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtProvider {
+
+    public static final String EMAIL = "email";
+
+    private final SecretKey secretKey;
+    private final long validityInMilliseconds;
+
+    public JwtProvider(@Value("${security.jwt.token.secret-key}") String key,
+      @Value("${security.jwt.token.expire-length}") long validityInMilliseconds) {
+        this.secretKey = Keys.hmacShaKeyFor(key.getBytes(StandardCharsets.UTF_8));
+        this.validityInMilliseconds = validityInMilliseconds;
+    }
+
+    public String createToken(String email) {
+        Date now = new Date();
+        Date validity = new Date(now.getTime() + validityInMilliseconds);
+
+        return Jwts.builder()
+          .claim(EMAIL, email)
+          .setIssuedAt(now)
+          .setExpiration(validity)
+          .signWith(secretKey, SignatureAlgorithm.HS256)
+          .compact();
+    }
+
+    public boolean isValid(String token) {
+        try {
+            Jws<Claims> claims = Jwts.parserBuilder()
+              .setSigningKey(secretKey)
+              .build()
+              .parseClaimsJws(token);
+
+            return !claims.getBody()
+              .getExpiration()
+              .before(new Date());
```

Comment:
리턴 값을 파악하기 쉽지 않네요😭. 메서드를 분리해서 적절한 네이밍을 주면 더 좋을 것 같아요.

Reply:
claims를 조회하는 부분을 따로 메서드 분리했습니다 :)

Reply:
앗 저는 return 값으로 활용하는 
```suggestion
            return isBeforeExpiration(claims)
```

이런 형식으로 예상되는 반환 값을 명확하게 표현해주는 건 어떨까? 라는 의도였습니다 🕺